saltychat/SaltyClient/External/Ped.cs

```diff
@@ -19,9 +19,11 @@ namespace RedM.External
 {
     public class Ped : Entity
     {
+        public Tasks Tasks { get; }
+
         public Ped(int handle) : base(handle)
         {
-
+            Tasks = new Tasks(this);
```

Comment:
I'm a big fan of `this` :)
```suggestion
            this.Tasks = new Tasks(this);
```